airbyte-db/lib/configs_db_schema.md

```diff
@@ -0,0 +1,361 @@
+
+| Column | Type  | Required | Default | Notes |
+| ------ | :---: | :---:    | :---:   | ---   |
+| notification_id | char(36) | yes | | Foreign key. Unique index to enforce 1:1 mapping. |
+| webhook | varchar(2048) | yes | | |
+
+### `SOURCE_DEFINITION`
+
+YAML reference:
+- [StandardSourceDefinition](https://github.com/airbytehq/airbyte/blob/master/airbyte-config/models/src/main/resources/types/StandardSourceDefinition.yaml)
+
+| Column | Type  | Required | Default | Notes |
+| ------ | :---: | :---:    | :---:   | ---   |
+| name | varchar(255) | yes | | |
+| docker_repository | varchar(255) | yes | | Index (ideally this should be unique, but we have duplicates for marketing purpose). |
+| docker_image_tag | varchar(50) | yes | | |
+| documentation_url | varchar(255) | yes | | |
+| icon | varchar(63) | | | |
+
+### `SOURCE_CONNECTION`
+
+YAML reference:
+- [SourceConnection](https://github.com/airbytehq/airbyte/blob/master/airbyte-config/models/src/main/resources/types/SourceConnection.yaml)
+
+| Column | Type  | Required | Default | Notes |
+| ------ | :---: | :---:    | :---:   | ---   |
+| name | varchar(255) | yes | |
+| source_definition_id | char(36) | yes | | Foreign key. Index. |
+| workspace_id | char(36) | yes | | Foreign key. Index. |
+| configuration | jsonb | yes | | Must be a valid json string. |
+| tombstone | boolean | | false | index | |
+
+### `DESTINATION_DEFINITON`
+
+YAML reference:
+- [StandardDestinationDefinition](https://github.com/airbytehq/airbyte/blob/master/airbyte-config/models/src/main/resources/types/StandardDestinationDefinition.yaml)
+
+| Column | Type  | Required | Default | Notes |
+| ------ | :---: | :---:    | :---:   | ---   |
+| name | varchar(255) | yes | | |
+| docker_repository | varchar(36) | yes | | Index (ideally this should be unique, but we have duplicates for marketing purpose). |
+| docker_image_tag | varchar(63) | yes | | |
+| documentation_url | varchar(255) | yes | | |
+| icon | varchar(63) | | | |
+
+### `DESTINATION_CONNECTION`
+
+YAML reference:
+- [DestinationConnection](https://github.com/airbytehq/airbyte/blob/master/airbyte-config/models/src/main/resources/types/DestinationConnection.yaml)
+
+| Column | Type  | Required | Default | Notes |
+| ------ | :---: | :---:    | :---:   | ---   |
+| name | varchar(255) | yes | |
+| destination_definition_id | char(36) | yes | | Foreign key. Index. |
+| workspace_id | char(36) | yes | | Foreign key. Index. |
+| configuration | jsonb | yes | | Must be a valid json string. |
+| tombstone | boolean | | false | index | |
+
+### `SYNC`
+
+YAML reference:
+- [StandardSync](https://github.com/airbytehq/airbyte/blob/master/airbyte-config/models/src/main/resources/types/StandardSync.yaml)
+- [NamespaceDefinitionType](https://github.com/airbytehq/airbyte/blob/master/airbyte-config/models/src/main/resources/types/NamespaceDefinitionType.yaml)
+
+The `sourceId` and `destination
```

Comment:
Did your sentence get cut off here?  The quotes are incomplete and it looks like it was trying to say more.

Reply:
Oh, this was no longer relevant. I have deleted it.